_tests/integration/helper.go

```diff
@@ -7,16 +7,32 @@ import (
 	"testing"
 	"time"
 
+	"github.com/bitrise-io/bitrise/log"
 	"github.com/stretchr/testify/require"
 )
 
+var binPathStr string
```

Comment:
This change makes it easier to run the integration tests locally and to iterate on the code. After a code change a new CLI version can be installed with `go build` and integration tests are ready to be run without setting `INTEGRATION_TEST_BINARY_PATH`.

Reply:
You are right, I meant `go install` not `go build`.